src/nncf/quantization/advanced_parameters.py

```diff
@@ -286,6 +286,22 @@ class AdvancedQuantizationParameters:
     # Backend specific parameters
     backend_params: dict[str, Any] = field(default_factory=dict)
 
+    # Backend parameter names
+    COMPRESS_WEIGHTS = "compress_weights"
+
+    def is_weight_compression_needed(self) -> bool:
+        """
+        Determine whether weight compression is needed based on advanced quantization parameters.
```

Comment:
Please move the function to the module scope, currently it is a method of advanced parameters

Reply:
Moved